README.md

```diff
@@ -113,138 +86,195 @@ You can still find the latest installers in the [24.9.2 release](https://github.
+| Windows | x86_64                        | Windows >= 7    | `Miniforge3-Windows-x86_64.exe` |
 
-Uninstalling Miniforge means removing the files that were created during the installation process.
-You will typically want to remove:
+`(*)` The Python version is specific only to the base environment. Conda can create new environments with different Python versions and implementations.
 
-1. Any modifications to your shell rc files that were made by Miniforge:
+`(**)` For Raspberry PI that include a 64 bit processor, you must also use
+a 64-bit operating system such as
+[Raspberry Pi OS 64-bit](https://www.raspberrypi.com/software/operating-systems/#raspberry-pi-os-64-bit)
+or
+[Ubuntu for Raspberry PI](https://ubuntu.com/raspberry-pi).
+The versions listed as "System: 32-bit" are not compatible with the installers on this website.
 
-```bash
-# Use this first command to see what rc files will be updated
-conda init --reverse --dry-run
-# Use this next command to take action on the rc files listed above
-conda init --reverse
-# Temporarily IGNORE the shell message
-#       'For changes to take effect, close and re-open your current shell.',
-# and CLOSE THE SHELL ONLY AFTER the 3rd step below is completed.
-```
+`(***)` Apple silicon builds are experimental and haven't had testing like the other platforms.
 
-2. Remove the folder and all subfolders where the base environment for Miniforge was installed:
 
-```bash
-CONDA_BASE_ENVIRONMENT=$(conda info --base)
-echo The next command will delete all files in ${CONDA_BASE_ENVIRONMENT}
-# Warning, the rm command below is irreversible!
-# check the output of the echo command above
-# To make sure you are deleting the correct directory
-rm -rf ${CONDA_BASE_ENVIRONMENT}
+## Install
+
+On all platforms, the installation should be finalized with commands like
+
+```sh
+conda init --all
 ```
 
-3. Any global conda configuration files that are left behind.
+(see `conda init -h`) and potentially also
 
-```bash
-echo ${HOME}/.condarc will be removed if it exists
-rm -f "${HOME}/.condarc"
-echo ${HOME}/.conda and underlying files will be removed if they exist.
-rm -fr ${HOME}/.conda
+```sh
+conda config --set auto_activate_base false
 ```
 
 ### Windows
 
-Download and execute the Windows installer. Follow the prompts, taking note of the options to
+Download and execute [the Windows installer](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe).
+Follow the prompts, taking note of the options to
 "Create start menu shortcuts" and "Add Miniforge3 to my PATH environment variable". The latter is
 not selected by default due to potential conflicts with other software. Without Miniforge3 on the
 path, the most convenient way to use the installed software (such as commands `conda` and `mamba`)
 will be via the "Miniforge Prompt" installed to the start menu.
 
+If desired, the
+`C:\Users\myusername\miniforge3\condabin` folder may be added to the path environment variable
```

Comment:
Note that for installation for all users, the path is of course different. It would be good to mention it.

Reply:
thank you.